src/test/java/org/elasticsearch/index/query/BaseQueryTestCase.java

```diff
@@ -160,7 +168,7 @@ public void testFromXContent() throws IOException {
     public void testToQuery() throws IOException {
         testQuery = createTestQueryBuilder();
         QueryParseContext context = createContext();
-        context.setMapUnmappedFieldAsString(true);
+        context.setAllowUnmappedFields(true);
```

Comment:
That must be a merge glitch, I think we settled in mapping fields to String by default for now as long as we don't need it otherwise. Will revert that.